frontend/app/validations/dependency-version.js

```diff
@@ -0,0 +1,33 @@
+import {
+  validatePresence,
+  validateLength,
+  validateFormat,
+  validateDate,
+} from 'ember-changeset-validations/validators';
+import validateOtherDate from 'outdated/validators/other-date';
+
+export default {
+  version: [
+    validatePresence({ presence: true, ignoreBlank: true }),
+    validateLength({ max: 100 }),
+    validateFormat({
+      regex:
+        /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/,
+      message: '{description} should be a valid semantic version',
+    }),
+    validateOtherDate({ after: 'releaseDate' }),
```

Comment:
I think the version should not be dependent to a date

Reply:
fixed in https://github.com/adfinis/Outdated/pull/14/commits/93d352b638a13daae76612f3f4bcc34666af412d